### DISCUSSION TITLE
请问纯dotnet项目怎么导入Puerts啊

### DISCUSSION THREAD

**Discussion by kay1314** [Tencent/puerts#1662] [Q&A]
想单纯作为C#调用ts的工具。按照官方说明引用源码文件会报错，官方给的donet的demo代码也不太全。

**Comment by chexiongsheng** [comment]
报什么错呢？puerts的ut之前就是在dotnet环境下跑的，现在已经添加了unity环境，但dotnet还是保留，每次提交都会编译执行，并没什么报错。

**Reply by kay1314** [reply]
会报puerts.dll找不到，官方给出的demo代码并没有完整donet项目引用

**Comment by chexiongsheng** [comment]
你可以参考下ut环境，我估计你是搞了unity专属的c#代码进工程，而且少定义了宏。

**Reply by kay1314** [reply]
有按照教程定义了宏
![image](https://github.com/Tencent/puerts/assets/80634200/02a091a4-06dc-40c4-a8a4-8346331ddb42)
教程说的“ 引用所有PuerTS C#代码 ”，指的是哪部分的代码

**Reply by chexiongsheng** [reply]
运行node ../../cli dotnet-test v8_9.4会生成出dotnet项目，然后你看下*.csproj文件或者用vs打开sln文件查看。
